docs/run.md

```diff
@@ -88,6 +88,8 @@ Below commands shows how to run Microsoft tier 0 (t0) tests on an Azure gallery
     }
     ```
 
+    For internal LISA V3 ramping up hands on, one should use the subscription "Linux Integration Services TEST ONLY" (id: 0cc2a67a-58b9-4e4f-98a8-bfa46a28e896). Contact lili.deng@ if not have access.
```

Comment:
It's internal information, shouldn't be in external repo. Please add it in our internal newcomer guideline.